LandSandBoat/server
# [SQL] Add some missing item_equipment and item_weapon entries. Fix some incorrect MIDs

With corrections to item_basic.sql
As per the previous PR request, single space between EOL and comment.



**_I affirm:_**

## What does this pull request do?
Adds missing items to item_equipment and item_weapon sql.

## Steps to test these changes

As GM, use command !additem itemid 1
See if the items spawn in inventory (Make sure inventory is not full when attempting).
If yes, make sure they can be equipped. If it is a weapon, make sure it changes the character damage value.
